src/sentry/services/hybrid_cloud/auth/impl.py

```diff
@@ -94,22 +94,17 @@ def get_user_ids(org_id: int, mem_id: int) -> Any:
                         organization_id=org_id
                     )
                     return (
-                        OrganizationMember.objects.filter(
+                        OrganizationMemberMapping.objects.filter(
                             Q(id__in=all_top_dogs_from_teams) | Q(role=roles.get_top_dog().id),
                             organization_id=org_id,
-                            user_is_active=True,
-                            user_id__isnull=False,
+                            user__is_active=True,
                         )
                         .exclude(id=mem_id)
                         .values_list("user_id")
                     )
 
                 if SiloMode.get_current_mode() != SiloMode.MONOLITH:
-                    # Giant hack for now until we have control silo org membership table.
-                    with SiloMode.exit_single_process_silo_context(), SiloMode.enter_single_process_silo_context(
-                        SiloMode.MONOLITH
-                    ):
-                        user_ids = get_user_ids(member.organization_id, member.id)
+                    user_ids = get_user_ids(member.organization_id, member.id)
```

Comment:
@RyanSkonnord you no longer need this if/else block if they're both the same now.

Reply:
Oh yeah, and that means we don't need `get_user_ids` as a helper function either.